src/backend/src/services/work-packages.services.ts

```diff
@@ -729,4 +729,15 @@ export default class WorkPackagesService {
 
     return workPackageTemplateTransformer(workPackage);
   }
+
+  static async getAllWorkPackageTemplates(submitter: User): Promise<WorkPackageTemplate[]> {
```

Comment:
we've been bad about it in this file, but you need to add a detailed comment to this service method